.github/workflows/update-semver.yml

```
@@ -0,0 +1,19 @@
+name: update-semver
+
+on:
+  release:
+    types: [published]
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  update-semver:
+    runs-on: ubuntu-18.04
```

I prefer `latest` b/c GH tends to deprecate some envs, meaning we'd have to re-hardcode them.
```suggestion
    runs-on: ubuntu-latest
```